packages/@react-stately/selection/src/types.ts

```diff
@@ -27,6 +27,7 @@ export interface SingleSelectionState extends FocusState {
 
 export interface MultipleSelectionState extends FocusState {
   selectionMode: SelectionMode,
+  disableEmptySelection?: boolean,
```

Comment:
Should this be in `SingleSelectionState` as well?

Reply:
added